packages/api-derive/test/e2e/promise.spec.ts

```diff
@@ -6,10 +6,10 @@ import ApiPromise from '@polkadot/api/promise/Api';
 import { BlockNumber } from '@polkadot/types';
 import { WsProvider } from '@polkadot/rpc-provider';
 
-// const WS = 'ws://127.0.0.1:9944/';
-const WS = 'wss://poc3-rpc.polkadot.io/';
+const WS = 'ws://127.0.0.1:9944/';
+// const WS = 'wss://poc3-rpc.polkadot.io/';
 
-describe.skip('derive e2e', () => {
+describe('derive e2e', () => {
```

Comment:
`.skip`

Reply:
I'll keep it in mind (and open as comment). 
But as long as the PR is WIP I'm not adding the skip.